exercises/rna-transcription/RnaTranscriptionExample.swift

```diff
@@ -1,20 +1,17 @@
 // Foundation not needed
 
-
-
 struct Nucleotide {
     
     var complementOfDNA:String{ return transcrive(DnaToRna) }
-    var complementOfRNA:String{ return transcrive(RnaToDna) }
     
-    var value = ""
+    private var value = ""
```

Comment:
Thanks! I am doing the editing on the web interface :) 
"Immutable value 'self.value' may only be initialized once"